client/my-sites/plugins/plugins-browser/index.jsx

```diff
@@ -355,7 +355,7 @@ export class PluginsBrowser extends Component {
 	}
 
 	shouldShowManageButton() {
-		if ( this.props.isJetpackSite ) {
+		if ( this.props.jetpackNonAtomic || this.props.hasBusinessPlan ) {
```

Comment:
This should be shown on Ecommerce plans too right? Maybe it would be a good use of the `isSiteOnAtomicPlan ` selector. It's poorly named, but it represents both Business and Ecomm sites. https://github.com/Automattic/wp-calypso/blob/38c37bbfe35b8c741c4e346637987ddc0fb3ab59/client/state/selectors/is-site-on-atomic-plan.js#L15

Reply:
Or would it work to replace them both with `canCurrentUser( state, selectedSite.ID, 'activate_plugins' );`?

Reply:
`hasBusinessPlan` [checks for](https://github.com/Automattic/wp-calypso/blob/trunk/client/my-sites/plugins/plugins-browser/index.jsx#L526) Business, Ecommerce, and Enterprise plans. I'm not familiar with the `isSiteOnAtomicPlan` selector, so I'll take a look at that.

Currently, `canCurrentUser` doesn't support the `activate_plugins` capability. We might be able to add that capability, and I would definitely like to investigate that in the future. I'm just not sure if there was a reason that it was left out.

Reply:
[`isSiteOnAtomicPlan`](https://github.com/Automattic/wp-calypso/blob/trunk/client/state/selectors/is-site-on-atomic-plan.js#L22) looks useful, but now I'm not sure what the Enterprise plan is and whether that needs to be checked as well 🤔